### PR TITLE
Navigation bar hover color, LeftPane button hover color

### PR DESCRIPTION
**Pull request by Veske**
* Navigation bar entries now give colored feedback when hovered over.
  * LeftPane buttons now give color feedback when hovered over.

### CODE REVIEW

File: src/main/resources/com/playonlinux/ui/impl/javafx/common/common.css [hunk 1]
```diff
@@ -78,6 +82,12 @@
     -fx-spacing: 0.4em;
 }
 
+.leftPaneButtons:hover {
+	-fx-background-color: #D3D3D3;
+	-fx-background-insets: 0,1,2;
+    -fx-background-radius: 3,2,1;
```
Comment (qparis) [inline src/main/resources/com/playonlinux/ui/impl/javafx/common/common.css]: Can you use only spaces?

**Comment by qparis** [comment]
Can you attach some screenshots? Thanks :)

File: src/main/resources/com/playonlinux/ui/impl/javafx/common/common.css [hunk 2]
```diff
@@ -78,6 +82,12 @@
     -fx-spacing: 0.4em;
 }
 
+.leftPaneButtons:hover {
+	-fx-background-color: #D3D3D3;
+	-fx-background-insets: 0,1,2;
+	-fx-background-radius: 3,2,1;
+}
```
Comment (qparis) [inline src/main/resources/com/playonlinux/ui/impl/javafx/common/common.css]: Please use only white spaces :)

**Comment by Veske** [comment]
https://www.dropbox.com/s/y4t1syxe2lqswkr/video.mp4?dl=0   Here is a video for demonstration. Mouse cursor recording seems to be bugged with Kazam.

**Comment by qparis** [comment]
ok to test

**Pull request merged**